pom.xml

```diff
@@ -94,6 +94,11 @@
             <ratchetFrom>origin/lts-2025</ratchetFrom>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-antrun-plugin</artifactId>
+          <version>3.1.0</version>
```

Comment:
Looks like everything working with `3.1.0` -> all ftests passing

Should we add this version to Nuxeo root pom or keep it here?

Reply:
We will add it back to the nuxeo root POM.